charts/aws-adapter/crds/security.nirmata.io_awsconfigs.yaml

```diff
@@ -0,0 +1,326 @@
+                  name:
+                    type: string
+                  networking:
+                    properties:
+                      ipFamily:
+                        type: string
+                      serviceIPv4CIDR:
+                        type: string
+                      serviceIPv6CIDR:
+                        type: string
+                      vpc:
+                        properties:
+                          clusterSecurityGroupID:
+                            type: string
+                          endpointPrivateAccess:
+                            type: boolean
+                          endpointPublicAccess:
+                            type: boolean
+                          publicAccessCIDRs:
+                            items:
+                              type: string
+                            type: array
+                          securityGroupIDs:
+                            items:
+                              type: string
+                            type: array
+                          subnetIDs:
+                            items:
+                              type: string
+                            type: array
+                          vpcID:
+                            type: string
+                        type: object
+                    type: object
+                  oidcProvider:
+                    type: string
+                  platformVersion:
+                    type: string
+                  region:
+                    type: string
+                  roleArn:
+                    type: string
+                  status:
+                    type: string
+                  tags:
+                    additionalProperties:
+                      type: string
+                    type: object
+                required:
+                - name
+                - region
+                - status
+                type: object
+              lastPollInfo:
+                properties:
+                  failure:
+                    properties:
+                      error:
+                        type: string
+                      message:
+                        type: string
+                    required:
+                    - error
+                    - message
+                    type: object
+                  status:
+                    type: string
+                  timestamp:
+                    format: date-time
+                    type: string
+                type: object
+              lastUpdatedTimestamp:
+                description: 'INSERT ADDITIONAL STATUS FIELD - define observed state
+                  of cluster Important: Run "make" to regenerate code after modifying
+                  this file'
+                format: date-time
+                type: string
+            required:
+            - lastPollInfo
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources:
+      status: {}
```

Comment:
Also, will be good to have comments for various CRD fields for someone going through it. Adding comments in the struct in *_types.go should achieve it.

Reply:
Will do that, thanks!

Reply:
Are we fixing the CRDs comment in this PR ?

Reply:
Hey @prateekpandey14, I'll be working on all the fixes including the vulnerability issue as separately mentioned by Ritesh for AWS adapter today. Sorry for the delay.